README.md

```diff
@@ -26,6 +26,31 @@ With a slight tweak of the y-axis label, core domain charts can be used to plan
 
 ![alt text](resources/architecture-migration-core-domain-chart.jpg "Architecture Migration Core Domain Chart")
 
+## Suggestions for Measuring Complexity and Differentiation
+
+Firstly, measuring complexity and differentiation of your domains is hard and is usually quite subjective. Strategy is a bet on the future, so you can never be sure exactly how complex or differentiating something will turn out to be. But there is still a lot of value in discussing and visualising your beliefs of the complexity and differentiation of each domain.
+
+The following clues can help to assess the complexity and differentiation of each domain. 
+ 
+- How difficult is it to design a conceptual model for the domain and build (and maintain) it as software? (essential domain complexity)
+- Is the current technical solution more complex than it needs to be to provide the current functionality? (accidental technical complexity)
+- Are there complex processes, calculations, and decisions happening outside the software? (operational complexity)
+- How hard would it be for a new entrant to the market to match or exceed the capability?
+- How hard would it be for an existing competitor to match or exceed the capability?
+- How much of an advantage is currently produced by the domain (revenue, brand, engagement)?
```

Comment:
What do you mean by "produced by the domain": our solution? I know that the repo is called core-domain-charts but this sentence is still somehow ambiguous for me. If I imagine that I use this chart to make my product/my solution proposal better and easier to evolve then I would never ask the question in this form. Maybe a small change like 
```suggestion
- How much of an advantage is currently produced by this business capability (revenue, brand, engagement)?
```
would remove this ambiguity?

Reply:
You're right, produced isn't the right word here used in this way. I will avoid business capability because that has a lot of connotations with other frameworks, but derived is a good fit. "How much of the company(s) competitive advantage is derived from the domain."

Let me know if you think we can improve it further.